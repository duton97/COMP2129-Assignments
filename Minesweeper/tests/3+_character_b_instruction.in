g 10 10
b 3 4
b 1 2 3
