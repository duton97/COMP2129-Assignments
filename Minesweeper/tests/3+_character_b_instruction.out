g 10 10
b 3 4
error
